### PR TITLE
 Add entry point script to atacworks

### PR DESCRIPTION
**Pull request by ntadimeti**
Add entry point script. 

Now atacworks can be run as follows:

`atacworks [train/denoise/eval] --input_files <> --intervals_file <> --sizes_file <> --weights_path <>`

### CODE REVIEW

**Comment by ntadimeti** [comment]
Changes to modularize the get_intervals and bw2h5 will be addressed as a separate PR

File: scripts/cmd_args.py [hunk 1]
```diff
@@ -198,10 +199,18 @@ def add_inference_options(parser):
...
+    parser.add('--input_files', required=True, type=str,
                help='list of data files in the form of "[file1, file2, '
                     '...]";'
                     'or a single path to a file or folder of files')
+    parser.add('--peaks', action='store_true',
+               help='Output denosied peaks from atacworks. If --task is regression, \
+                       model only outputs denoised tracks and \
+                       this option becomes irrelevant.')
+    parser.add('--tracks', action='store_true',
+               help='Output denosied tracks from atacworks. If --tasj is classification, \
```
Comment (tijyojwad) [inline scripts/cmd_args.py:211]: typo: `tasj` -> `task`

File: tests/end-to-end/train.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ check_status $? "Training run not succesful!"
 echo ""
 echo "Test classification mode of training"
 echo ""
-python $root_dir/main.py train \
+atacworks train \
```
Comment (tijyojwad) [inline tests/end-to-end/train.sh:31]: do any of the READMEs or ipython notebooks need to be updated with this as well?

Comment (ntadimeti) [reply tests/end-to-end/train.sh:31]: updated the notebooks. they were merged this morning :)

File: scripts/main.py [hunk 3]
```diff
@@ -175,12 +177,22 @@ def writer(infer, intervals_file, exp_dir, result_fname,
...
+
+    if task == "both":
+        if tracks and not peaks:
+            channels = [0]
+        elif peaks and not tracks:
+            channels = [1]
+        elif tracks and peaks:
+            channels = [0, 1]
+        # If both tracks and peaks are false, turn them to default true.
+        elif not(tracks and peaks):
```
[the remark this answers is not in the document]
Comment (ntadimeti) [reply scripts/main.py]: Yep, good catch!

File: scripts/cmd_args.py (hunk 1 above)
Comment (ntadimeti) [reply scripts/cmd_args.py]: done

**Pull request merged**